federated-css/e2e/helpers.ts

```diff
@@ -0,0 +1,7 @@
+import {
+  selectors,
+} from '../../cypress/common/selectors';
+
+export function getE2eDataElementByNumber(n: number): string {
+  return `${selectors.dataE2e}:nth-child(${n})`;
```

Comment:
This could be reworked if you will create proper e2e selector, also it belongs to sample->e2e->methods, check as we did it before with methods.ts